pkgs/applications/editors/neovim/utils.nix

```diff
@@ -13,46 +13,11 @@
 let
   inherit (vimUtils) toVimPlugin;
 
-   /* returns everything needed for the caller to wrap its own neovim:
-   - the generated content of the future init.vim
-   - the arguments to wrap neovim with
-   The caller is responsible for writing the init.vim and adding it to the wrapped
-   arguments (["-u" writeText "init.vim" GENERATEDRC)]).
-   This makes it possible to write the config anywhere: on a per-project basis
-   .nvimrc or in $XDG_CONFIG_HOME/nvim/init.vim to avoid sideeffects.
-   Indeed, note that wrapping with `-u init.vim` has sideeffects like .nvimrc wont be loaded
-   anymore, $MYVIMRC wont be set etc
-   */
-  makeNeovimConfig =
-    { withPython3 ? true
-    /* the function you would have passed to python3.withPackages */
-    , extraPython3Packages ? (_: [ ])
-    , withNodeJs ? false
-    , withRuby ? true
-    /* the function you would have passed to lua.withPackages */
-    , extraLuaPackages ? (_: [ ])
```

Comment:
What's the new expected way of handling adding extraLuaPackages to a config? I didn't see this argument moved to the wrapper like extraPython3Packages was.

Reply:
In relation to https://buildbot.nix-community.org/#/builders/2809/builds/1218 trying to figure out what we need to change in https://github.com/nix-community/nixvim/blob/0ca98d02104f7f0a703787a7a080a570b7f1bedd/modules/top-level/output.nix#L266

Reply:
you are absolutely correct. I would like to avoid adding them to the wrapper as extraLuaPackages was a workaround to specify plugin dependencies. Lua dependencies are now loaded correctly in the wrapper via shell hooks. A user could call the wrapper directly adding its own wrapper args to add externmal lua dependencies or convert a lua package with `buildNeovimPlugin`

Reply:
if you could trigger the nnixvim job again, hopefully it is now fixed.

Reply:
Cool, kicking off another build there